Add Padding (#37340)
demisto/content: Packs/MicrosoftGraphMail/ReleaseNotes/1_6_20.md

#### Integrations

##### Microsoft Graph Mail Single User

> Fixed an issue where the ***send-mail***, ***reply-mail***, ***get-attachments*** and ***fetch-incidents*** commands failed to decode the **htmlBody** argument if it was provided with no padding.

Reviewer: ```suggestion
Fixed an issue where the following commands failed to decode the **htmlBody** argument if it was provided with no padding:
- ***send-mail***
- ***reply-mail***
- ***get-attachments***
- ***fetch-incidents*** 
```

##### O365 Outlook Mail (Using Graph API)

> Fixed an issue where the ***send-mail***, ***reply-mail***, ***get-attachments*** and ***fetch-incidents*** commands failed to decode the **htmlBody** argument if it was provided with no padding.

Reviewer: ```suggestion
Fixed an issue where the following commands failed to decode the **htmlBody** argument if it was provided with no padding:
- ***send-mail***
- ***reply-mail***
- ***get-attachments***
- ***fetch-incidents*** 
```

#### Integrations

##### Microsoft Graph Mail Single User

> Fixed an issue where the following commands failed to decode the **htmlBody** argument if it was provided with no padding:

Reviewer 2: Same as above:
You fixed also comments that do not use the htmlBody such as file_result_creator inside the create_attachment command (also would suggest changing the get-attachments to the real name), maybe i would change the release notes to :
Fixed an issue where the following commands failed to decode Base64-encoded attachments or images inside the htmlBody when provided without padding:

send-mail
reply-mail
msgraph-mail-get-attachment
fetch-incidents
wdyt?

- ***send-mail***
- ***reply-mail***
- ***msgraph-mail-get-attachment***
- ***fetch-incidents*** 

##### O365 Outlook Mail (Using Graph API)

Fixed an issue where the following commands failed to decode the **htmlBody** argument if it was provided with no padding:
- ***send-mail***
- ***reply-mail***

> - ***get-attachments***

Reviewer 2: ```suggestion
- ***msgraph-mail-get-attachment***
```

> Fixed an issue where the following commands failed to decode Base64-encoded attachments or images inside the html נody when provided without padding:
> - ***send-mail***
> - ***reply-mail***
> - ***msgraph-mail-get-attachment***
> - ***fetch-incidents*** 
> 
> ##### O365 Outlook Mail (Using Graph API)
> 
> Fixed an issue where the following commands failed to decode the **htmlBody** argument if it was provided with no padding:
> - ***send-mail***
> - ***reply-mail***
> - ***get-attachments***
> - ***fetch-incidents*** 

Author: ```suggestion
Fixed an issue where the following commands failed to decode Base64-encoded attachments or images inside the html body when provided without padding:
- ***send-mail***
- ***reply-mail***
- ***msgraph-mail-get-attachment***
- ***fetch-incidents*** 

##### O365 Outlook Mail (Using Graph API)

Fixed an issue where the following commands failed to decode Base64-encoded attachments or images inside the html body when provided without padding:
- ***send-mail***
- ***reply-mail***
- ***msgraph-mail-get-attachment***
- ***fetch-incidents*** 
```
